app/assets/stylesheets/govuk_publishing_components/components/print/_feedback.scss

```diff
@@ -0,0 +1,3 @@
+.pub-c-feedback {
+  display: none;
```

Comment:
I'm not sure how print styles work... But if I print the component guide page, I think the component still shows (just without styling)

Reply:
Yeah, it does, which is weird. Looks like the print stylesheet is working (works for the step nav component) but not this one. Been staring at this for ages and can't figure it out. Will keep going.

Reply:
Ah, figured it out. It was in the print stylesheet for the components, but not the print stylesheet for the component guide. Obviously.